src/main/java/com/codeday/productivity/entity/Goal.java

```diff
@@ -0,0 +1,55 @@
+package com.codeday.productivity.entity;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Column;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.Instant;
+
+@Entity
+@Table(name = "GOAL_TBL")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Goal {
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    private String title;
+
+    private String description;
+
+    @Column(columnDefinition = "TIMESTAMP")
+    private Instant startDate;
+
+    @Column(columnDefinition = "TIMESTAMP")
+    private Instant endDate;
+
+    @Column(name = "due_date", columnDefinition = "TIMESTAMP")
```

Comment:
How is due date different from end date?

Reply:
We made this design change in our discussion with the team. Due date indicates when the goal is expected to be completed, while the end date signifies the actual date the goal was concluded or terminated.